wiki/Tournaments/RMoT_Invitational/5/en.md

```diff
@@ -0,0 +1,166 @@
+- ![][flag_US] [jppz](/users/7889571 "jppz")
+- ![][flag_AU] [Klein-](/users/2153933 "Klein-")
+- ![][flag_RU] [KoloMl](/users/6544403 "KoloMl")
+- ![][flag_RU] [Nogad](/users/6488077 "Nogad")
+- ![][flag_GB] [Olii](/users/7683305 "Olii")
+- ![][flag_BR] [PlayPen](/users/5094351 "PlayPen")
+- ![][flag_FR] [Prosdokies](/users/3852436 "Prosdokies")
+- ![][flag_RU] [Well Halen](/users/8660244 "Well Halen")
+- ![][flag_UA] [Xelite](/users/7793117 "Xelite")
+
+**Notice:** Klein- and Nogad gave up their places in the Invitational, and iTzAshraf and \_Suzuha reached semi-finals twice. Due to this, 4 people who have not reached semi-finals were selected to participate in the tournament: filler players.
+
+The filler players are chosen according to a few criteria in the following priority:
+
+1. Having participated in and reached QF (quarter-finals) once out of the last 4 RMoT's
+2. Number of times reached QF
+3. Number of times participated in RMoT
+4. Current PP
+
+![][flag_MY] [Aasheda\_](/users/7614055 "Aasheda\_"), ![][flag_US] [Likean00b](/users/4860447 "Likean00b"), ![][flag_PL] [My Angel Erwin](/users/6814521 "My Angel Erwin") and ![][flag_RU] [Skif4a8929](/users/6700953 "Skif4a8929") gained the right to participate in the Invitational, ![][flag_RU] [Skif4a8929](/users/6700953 "Skif4a8929") being seeded lower than the other three.
+
+## Rules
+
+### General Rules
+
+- The tournament will be held in 1v1 double-elimination format with traditional seeding.
+- The matches will be played using Score V2 and TeamVS settings.
+- A referee will be present in every match and every match will be streamed, with additional commentators when possible.
+- The tournament will be held over three weekends, joining the [Discord server](https://discord.gg/9sKe7nF "Discord invite link") is mandatory at least for the duration of the tournament.
+- Round of 16 and Losers Round 1 are BO5, Quarter Finals, Semi-Finals, Losers Round 2, and Losers Round 3 are BO7. Finals, Losers Round 4, and Losers Semi-Finals are BO9. Grand Finals and Losers Finals are BO11.
+
+### Match Rules
+
+1. The referee will create the room and invite or move the players in.
+2. The players will roll to determine the order of picking warmups and maps.
+3. The higher roller will pick the first warmup map.
+4. The lower roller will ban first, the higher roller will pick first.
+5. FreeMod is forced mod, meaning every player has to use at least one mod.
+6. Tiebreaker is not forced mod but you may choose a mod.
+7. Eligible mods for FreeMod and Tiebreaker are Easy, Flashlight, Hard Rock and Hidden.
+
+## Mappool
+
+There is only one mappool for all matches, made of beatmaps from the last four RMoT mappools. There are six tiebreaker beatmaps, one for each stage. Every tiebreaker from the last four RMoT's and two new beatmaps are picked for tiebreaker.
+
+[Download the mappack here!](http://www.mediafire.com/file/2telo31u7xasasn/RMoT%20Invitational%20%235%20-%20Mappool.zip)
```

Comment:
On other tournaments, the mappacks are bolded and have their file size in parenthesis `(182.04 MB)`.

You can just copy this.
`**[Download the mappack here! (182.04 MB)](http://www.mediafire.com/file/2telo31u7xasasn/RMoT%20Invitational%20%235%20-%20Mappool.zip)**`